core/domain/story_domain.py

```diff
@@ -598,13 +614,14 @@ def to_dict(self):
         }
 
     @classmethod
-    def create_default_story(cls, story_id):
+    def create_default_story(cls, story_id, title=feconf.DEFAULT_STORY_TITLE):
```

Comment:
Don't need a default value for the arg. Just require it to be passed in.

Reply:
Done.